pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -13,24 +13,20 @@
 assert withQt5 -> useQt4 == false;
 assert useQt4 -> withQt5 == false;
 
-with stdenv.lib;
-
 let
-  os = stdenv.lib.optionalString;
-  majorVersion = "3.13";
-  minorVersion = "4";
-  # from https://cmake.org/files/v3.13/cmake-3.13.4-SHA-256.txt for cmake-3.13.4.tar.gz
-  sha256 = "fdd928fee35f472920071d1c7f1a6a2b72c9b25e04f7a37b409349aef3f20e9b";
-  version = "${majorVersion}.${minorVersion}";
+  lib = stdenv.lib;
+  os = lib.optionalString;
+  version = "3.14.5";
```

Comment:
`version` and `sha256` don't need to be up here anymore, right?

Reply:
right. I went ahead and removed the `os` shortcut as well.